.github/workflows/pull-request-check.yaml

```
@@ -0,0 +1,46 @@
+name: Pull Request Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Run lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'
```

node.jsのversionの一貫性を保つために、package.jsonにnode.jsのversionを定義する感じにしたいです。

```suggestion
        with:
          node-version-file: package.json
```

---

バージョンのアップデートによる潜在的なリスクの予防のために、`v22.15.0`とpatch versionまでカッチリ定義した方が良いと考えています。

[Node.js — Node.js Releases](https://nodejs.org/en/about/previous-releases#looking-for-the-latest-release-of-a-version-branch)

---

ありがとうございます。package.json側でversionを設定します。